tiatoolbox/models/architecture/vanilla.py

```diff
@@ -84,7 +95,10 @@ def _get_timm_architecture(
     *,
     pretrained: bool,
 ) -> list[nn.Sequential, ...] | nn.Sequential:
-    """Get architecture and weights for pathology-specific timm models.
+    """Retrieve a timm model architecture.
+
+    This function fetches a model architecture from the timm library, specifically for
+    pathology-related tasks.
 
     Args:
         arch_name (str):
```

Comment:
Can you make it consistent with line 33 (_get_architecture)? 

arch_name (str):
            Name of the architecture (e.g. 'UNI', [ADD MORE IF NECESSARY]).

Reply:
Done, thanks Shan.